src/components/pages/home.jsx

```diff
@@ -112,15 +112,13 @@ const SearchResults = React.createClass({
     return (
       <List items={items} type="unstyled">
         {parameterOrVariable => {
-          const {name, type} = parameterOrVariable
-          const description = type === 'parameter'
-            ? parameterOrVariable.description
-            : parameterOrVariable.label
+          const {description, label, name} = parameterOrVariable
+          const displayedDescription = description || label
           return (
-            <Link key={`${name}-${type}`} to={`/${name}`}>
+            <Link key={`${name}`} to={`/${name}`}>
```

Comment:
Haha, this is dumb indeed. But after more thought I think this change was a mistake. It's *possible* as we said, that a variable and a parameter have the same name. That's why I added a `.type == 'variable' (or 'parameter')` attribute.

I must fix that.